clang/test/CodeGenSYCL/intel-fpga-ivdep-global.cpp

```diff
@@ -75,6 +75,40 @@ void ivdep_conflicting_safelen() {
   }
 }
 
+// Global ivdep w/ safelen value 1 is specified - do not annotate GEP
+//
+// CHECK: define {{.*}}spir_func void @_Z{{[0-9]+}}ivdep_safelen_onev()
+void ivdep_safelen_one() {
+  // CHECK: %[[ARRAY_A:[0-9a-z]+]] = alloca [10 x i32]
+  int a[10];
+  // CHECK: %[[ARRAY_B:[0-9a-z]+]] = alloca [10 x i32]
+  int b[10];
+  [[intel::ivdep(1)]] for (int i = 0; i != 10; ++i) {
+    // CHECK:  %{{[0-9a-z]+}} = getelementptr inbounds [10 x i32], ptr addrspace(4) %[[ARRAY_A]].ascast, i64 0, i64 %{{[0-9a-z]+}}
```

Comment:
Can you explain if/why you need both arrays here? 

Also, wouldn't this test pass if GEP was annotated as well? To check annotation does not happen, I think we need to add a CHECK-NOT for `llvm.index.group` right?

Reply:
Thanks @elizabethandrews for reviews. 

>>Can you explain if/why you need both arrays here?

In this file, all test cases are using both arrays, so I added both array. There is no need here for both arrays. I have removed one.

>>Also, wouldn't this test pass if GEP was annotated as well? T

Not sure. I understand your question.

>>To check annotation does not happen, I think we need to add a CHECK-NOT for llvm.index.group right?

Test is currently passing without CHECK-NOT. I have added CHECK-NOT to show annotation does not happen.

Reply:
Sorry for not being clear. I meant 

`// CHECK: %{{[0-9a-z]+}} = getelementptr inbounds [10 x i32], ptr addrspace(4) %[[ARRAY_A]].ascast, i64 0, i64 %{{[0-9a-z]+}}`

would pass even if annotation was happening and so a CHECK-NOT is required to accurately test that an annotation is not happening.

Reply:
I see. Thanks @elizabethandrews for clarification. 

>>would pass even if annotation was happening and so a CHECK-NOT is required to accurately test that an annotation is not happening.

Agreed.